foreign_cc/private/framework/BUILD.bazel

```diff
@@ -1,7 +1,13 @@
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 
+toolchain_type(
+    name = "toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
```

Comment:
I feel there is some value in having the word 'shell' in the Label for this as this Label will be visible to users even though its in the private directory e.g. with toolchain resolution debugging turned on.

Reply:
I feel like the package name would be representative of the toolchain type. But I can rename the actual type